### ISSUE TITLE
Surfacebook 3 using fwupdmgr returns No supported devices found

### ISSUE THREAD

**Issue by razlani** [linux-surface/surface-uefi-firmware#19]
As per title, albeit I'm attempting to use the same UEFI driver that worked (SurfaceBook2_SurfaceUEFI_390.3192.768.0.cab from SurfaceBook2_Win10_17763_20.061.12854.0.msi) for my surfacebook 2, on a surfacebook 3.

I can't seem to get the older SB3 firmware on the MS site, although I can see the updates history.

The UEFI should be the same anyway - is there no way to "force" the install, perhaps by massaging the metadata or something? I'm doing a 
`fwupdmgr install --allow-older --allow-reinstall SurfaceBook2_SurfaceUEFI_390.3192.768.0.cab`

Thanks a bunch!

**Comment by StollD** [comment]
This won't work. SB3 and SB2 are different hardware, in the same casing. Why do you want to do this?

**Comment by razlani** [comment]
> This won't work. SB3 and SB2 are different hardware, in the same casing. Why do you want to do this?

Thanks for the speedy response. 

Solely for the UEFI driver, to be able to undervolt since I know that driver was of a version permitting me to do that, and since I couldn't find sb3 historical firmware on a cursory search.

Is the solution to look for the equivalent sb3 firmware then? I assumed they'd be rocking the same UEFI and was willing to take that risk 😝

**Comment by StollD** [comment]
> Is the solution to look for the equivalent sb3 firmware then?

Yes, but I am finding reports about this from 2 years ago, so you would have to look far back. Maybe you can find someone that has a backup of the MSI from that time. If you do find one, remember to flash back all the firmwares, not just the UEFI, as the old UEFI could be incompatible with the newer firmwares.

**Comment by razlani** [comment]
Damn.
wayback machine only goes so far back as aug 2020 https://web.archive.org/web/*/https://www.microsoft.com/en-us/download/details.aspx?id=101315 - I needed it to be a few months sooner. 

Ah well, mayhaps some kind stranger comes along with some firmware after all
